app/src/components/multistepformpage.js

```diff
@@ -0,0 +1,7 @@
+export default function MultiStepFormPage({children, title}) {
+  return (
+    <div>
+      {children}
+    </div>
```

Comment:
Is the div necessary here?

Reply:
React components need a parent. However it can be replaced with <> </> (a null parent). I can change that.